src/main/java/seedu/address/ui/EventCard.java

```diff
@@ -75,7 +75,9 @@ public EventCard(Event event, int displayedIndex) {
         attendeesList.stream()
                 .forEach(attendee -> {
                     int index = attendeesList.indexOf(attendee) + 1;
-                    attendees.getChildren().add(new Label(index + " " + attendee.getName().toString()));
+                    attendees.getChildren().add(new Label(
+                            index + " " + attendee.getName() + " ("
+                            + attendee.getPhone() + ")"));
```

Comment:
The UI breaks when the phone number is very long, perhaps we can set a maximum length for phone numbers. 

<img width="1552" alt="image" src="https://user-images.githubusercontent.com/84223259/198826825-1eec5770-2285-4e22-aa33-a1452de8dc36.png">